include/integratorxx/quadratures/gausscheby2.hpp

```diff
@@ -72,6 +72,6 @@ struct quadrature_traits<GaussChebyshev2<PointType, WeightType>> {
 
     return std::make_tuple(points, weights);
   }
-}
+};
```

Comment:
Is this semicolon necessary? Its use was inconsistent between gaysscheby1 and gausscheby2.

Reply:
It closes `struct quadrature_traits<GaussChebyshev2<...>>`, so it fails to compile otherwise